bin/bulk_recalculations_check_species_names.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
I would get rid of this if we are not using it. Or at least clarify that we are not using it.

Reply:
Done